### PR TITLE
Update international-journal-of-geriatric-psychiatry.csl

### PR DESCRIPTION
**Pull request by POBrien333**
Found a whole bunch of mistakes just by random. Should be a bit better now.

### CODE REVIEW

File: international-journal-of-geriatric-psychiatry.csl [hunk 1]
```diff
@@ -45,6 +45,7 @@
   <macro name="author-short">
     <names variable="author">
       <name form="short" and="text" delimiter=", " delimiter-precedes-last="never" initialize-with=". "/>
+      <et-al/>
```
Comment (adam3smith) [inline international-journal-of-geriatric-psychiatry.csl]: why are you adding this? This shouldn't change anything.

File: international-journal-of-geriatric-psychiatry.csl [hunk 2]
```diff
@@ -20,7 +19,7 @@
     <issn>0885-6230</issn>
     <eissn>1099-1166</eissn>
     <summary>Harvard (author-date style) for International Journal of Geriatric Psychiatry</summary>
```
Comment (adam3smith) [inline international-journal-of-geriatric-psychiatry.csl]: we should add a note that this follows published articles and not the (useless) author guidelines

File: international-journal-of-geriatric-psychiatry.csl [hunk 3]
```diff
@@ -160,7 +159,7 @@
       </group>
     </layout>
   </citation>
-  <bibliography>
+  <bibliography delimiter-precedes-et-al="always" et-al-min="4" et-al-use-first="3">
```
Comment (adam3smith) [inline international-journal-of-geriatric-psychiatry.csl]: I think this is 6/3 or 7/3 I see at least 5 listed authors here: http://onlinelibrary.wiley.com/doi/10.1002/gps.4447/epdf I think 7/3 is the most common rule, so I'd go with that.

File: international-journal-of-geriatric-psychiatry.csl [hunk 4]
```diff
@@ -146,7 +145,7 @@
       </if>
     </choose>
   </macro>
-  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+  <citation delimiter-precedes-et-al="never" et-al-min="4" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" collapse="year">
```
Comment (adam3smith) [inline international-journal-of-geriatric-psychiatry.csl]: Looking at the Crellin et al. reference in http://onlinelibrary.wiley.com/doi/10.1002/gps.4447/epdf I think this should be 3/1

File: international-journal-of-geriatric-psychiatry.csl (hunk 1 above)
Comment (POBrien333) [reply international-journal-of-geriatric-psychiatry.csl]: Not sure where that is coming from. thanks for spotting it.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**